src/SFML/Window/Win32/WglContext.hpp

```diff
@@ -27,13 +27,23 @@
 ////////////////////////////////////////////////////////////
 // Headers
 ////////////////////////////////////////////////////////////
+#include <SFML/Window/Context.hpp>
 #include <SFML/Window/GlContext.hpp>
 
+#include <SFML/System/Vector2.hpp>
+
 #include <glad/wgl.h>
 
 
+namespace sf
+{
+struct ContextSettings;
+}
```

Comment:
Would nested namespaces make sense? Instead of opening `sf` here and `sf::priv` below